lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -487,10 +489,16 @@ def ensure_ok(self):
             # Zones and listeners will be added at creation
             self._create_elb()
         else:
-            self._set_zones()
-            self._set_security_groups()
-            self._set_elb_listeners()
-            self._set_subnets()
+            new_scheme = self._get_scheme()
```

Comment:
Also, is it possible for you to modify [this](https://github.com/ansible/ansible/pull/22916/files#diff-e4db2ae81c34b941c6d13e19825f24ebR1264) to pull the available schemes from botocore and validate, instead of just allowing any string?